sdk/datafactory/Azure.ResourceManager.DataFactory/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 ### Breaking Changes
 
 ### Bugs Fixed
+- Fix the issue: https://github.com/Azure/azure-rest-api-specs-examples/issues/3219
```

Comment:
Remove the issue link, add the description as you did in [another PR](https://github.com/Azure/azure-sdk-for-net/pull/37978/files/1ef206870f57687b3e71c9322e95b13c83be4a4e#diff-8f7e29015e546aacf4ba4f8493ddb58a9f20606c533dcf76e4e34c065fc117c4)

Reply:
fixed